packages/terra-table/src/terra-dev-site/doc/example/TableHeader.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import Table, { Header, HeaderCell } from 'terra-table/lib/index';
+/* eslint-enable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+const TableHeaderExample = () => (
+  <Table header={(
+    <Header>
+      <HeaderCell style={{ height: '50px' }} />
```

Comment:
It might be better to add some mock content that custom styles?

Reply:
Updated.